src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Lottery().getRandomBall();
+            System.out.println(ball);
+        }
+
         // create three balls using class Lottery and print information about them in console
     }
```

Comment:
1. Please avoid creation of unnecessary objects. We don't need to create 3 Lotteries, it's enough to create one and call method 3 times from same object
2. Please remove outdated comments in code

```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
    }
```

Reply:
я правильно понимаю, что если мы создаем 1 обьект и 3 раза вызываем метод, то он 3 раза меняет значение полей обьекта. тоесть у нас в итоге останется обьект с последней парой значения цвета и числа?

Reply:
Мы создаём объект лотереи, который на каждый вызов метода создаёт новый объект мяча. Поэтому он не меняет поля, он каждый раз отдаёт новый мяч.